drivers/sensor/lps22hh/lps22hh_i2c.c

```diff
@@ -0,0 +1,52 @@
+/* ST Microelectronics LPS22HH pressure and temperature sensor
+ *
+ * Copyright (c) 2019 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Datasheet:
+ * https://www.st.com/resource/en/datasheet/lps22hh.pdf
+ */
+
+#include <string.h>
+#include <i2c.h>
+#include <logging/log.h>
+
+#include "lps22hh.h"
+
+#ifdef DT_ST_LPS22HH_BUS_I2C
+
+static u16_t lps22hh_i2c_slave_addr = DT_INST_0_ST_LPS22HH_BASE_ADDRESS;
```

Comment:
This should be in `struct lps22hh_config`

Reply:
moved in config